staging/src/k8s.io/component-base/metrics/prometheus/restclient/metrics.go

```diff
@@ -103,6 +104,17 @@ var (
 			},
 		},
 	)
+
+	execPluginCalls = k8smetrics.NewCounterVec(
+		&k8smetrics.CounterOpts{
+			Name: "rest_client_exec_plugin_call_total",
+			Help: "Number of calls to an exec plugin, partitioned by the type of " +
+				"event encountered (plugin_called, plugin_not_found, " +
+				"client_internal_error) and an optional exit code which is set with " +
+				"plugin_called.",
+		},
+		[]string{"code", "call_status"},
```

Comment:
is call_status a good description for this label, or would `error_type` be clearer (and set to `no_error` when err is nil)?

Reply:
i went back and forth with myself on `call_status` vs `error_type`. i think i landed on `call_status` since it better matched the metric name (`rest_client_exec_plugin_*call*_total`) and so we could get away with 3 enum values for this field instead of 4. i figured that if someone wanted to filter down to all the unsuccessful executions, they could just say `exitCode != 0 && call_status == plugin_called`.

i'm not totally wed to this. i'll switch it over to `no_error` and just let me know if you change your mind: 9e947251851a1b44ae4c7c11457932cf151b1467